include/command-line-options.php

```diff
@@ -1921,6 +1921,13 @@ function. All achievable slopes are zero or positive.</p>
      <td>Returns the accumulator image in addition to the lines image</td>
   </tr>
 
+  <tr>
+    <td>heic:preserve-orientation</td>
+    <td>Preserve the original EXIF orientation during HEIC decoding and rotate the pixels accordingly.
+        By default, EXIF orientation is reset to "1" to match the actual orientation of pixels in HEIC
```

Comment:
This sentence is missing a dot at the end 😛

Reply:
Argh :-D
I will add it)

Reply:
By the way, 'hough-lines:accumulator=true' several lines above does not have the trailing dot too. I will add it applying a boyscout rule :)